accessibility.md

```diff
@@ -1,19 +1,26 @@
 ---
-layout: default
+layout: blue_page
 permalink: /accessibility/
 title: Accessibility Statement
 ---
 
-<div class="popout">
-  <section>
-      <h1>Accessibility Statement</h1>
-  </section>
-</div>
+> You may also be interested in the [**venue accessibility statement**](http://www.focusterra.ethz.ch/en/your-visit.html).
 
-<section markdown="1">
+RustFest is an inclusive conference and as such strives to be accessible to everyone. This includes:
 
-* The venue is wheelchair accessible, the toilet too!
-* There will be a a quiet room on Saturday, wheelchair accessible.
-* Please do not wear scents.
+* Offering significantly discounted tickets for concession holders.
+* Ensuring our venues are accessible by wheelchair.
+* Make sure that maps and orientation material is available.
+* Providing clear signage to, from and through the building
+* Providing quiet spaces for those with sensory issues.
+* Providing accessibility for conference sessions.
+* Providing complimentary tickets for assistants for those with disabilities.
+* Access to non-gendered toilets.
```

Comment:
Unfortunately my connection dropped around that time at the meeting at that point as well, so I might have missed something (?) - but my plan was (and still is) to ask :)